v03_pipeline/lib/misc/callsets.py

```diff
@@ -49,18 +51,34 @@ def callset_project_pairs(
     project_guids: list[str],
     project_remap_paths: list[str],
     project_pedigree_paths: list[str],
+    imputed_sex_paths: list[str] | None,
```

Comment:
if this logic works for callset-wide callset (duh) pairs , it should absolutely work for callset-wide predicted sex metrics yea ?

Reply:
I think so!